### ISSUE TITLE
3.0 example serialization

### ISSUE THREAD

**Issue by waltkb** [SMILEY4/ktor-openapi-tools#106]
An example like this:

```kotlin
example("E-mail + password") {
    value = buildJsonObject {
        put("type", JsonPrimitive("email"))
        put("email", JsonPrimitive("user@email.com"))
        put("password", JsonPrimitive("password"))
    }
}
```

Does not come out as a simple JSON like this:

```json
{
  "type": "email",
  "email": "user@email.com",
  "password": "password"
}
```

But instead produces this example:
```json
{
  "type": {
    "content": "email",
    "string": true
  },
  "email": {
    "content": "user@email.com",
    "string": true
  },
  "password": {
    "content": "password",
    "string": true
  }
}
```
![screenshot_20240613_145119](https://github.com/SMILEY4/ktor-swagger-ui/assets/68587968/25455cea-57b6-45b1-80bf-3e9eb588061c)

**Comment by waltkb** [comment]
Could probably be fixed with #107

**Comment by SMILEY4** [comment]
Hi,
i'm not quite sure where `buildJsonObject` comes from or what type it exactly returns, but you should be able to provide the example as an json string here.

```kotlin
example("E-mail + password") {
    value = buildJsonObject {
        put("type", JsonPrimitive("email"))
        put("email", JsonPrimitive("user@email.com"))
        put("password", JsonPrimitive("password"))
    }.toJsonString() // ... or something that turns the result of buildJsonObject into a json-string
}
```

I think this could then be done automatically  with the config provided with https://github.com/SMILEY4/ktor-swagger-ui/pull/107 so you dont need to convert to string for every individual example.
